test/integration/backward_compatible/parallel/security/SecurityTest.js

```diff
@@ -84,14 +100,24 @@ describe('SecurityTest', function () {
         let cluster;
         let member;
         const testFactory = new TestUtil.TestFactory();
+        let tokenEncoding;
 
         before(async function () {
             TestUtil.markEnterprise(this);
-            TestUtil.markClientVersionAtLeast(this, '5.1');
+            // Security config is added in 5.1.0, 4.2.1 and 5.0.3.
+            // Security config is added in 5.1.0, 4.2.1 and 5.0.3.
```

Comment:
I guess no need for the duplicated comment?

Reply:
ok there is another one below. Fixed both at bb84c238